src/gui/MainWindow.cpp

```diff
@@ -765,24 +765,24 @@ void MainWindow::openFile(const QString& filePath) {
 }
 
 bool MainWindow::saveFile(const QString& filePath) {
-    if (activeDocumentId == -1) return false;
     return documents[activeDocumentId]->getDatabase()->Save(filePath.toUtf8().data());
 }
 
-void MainWindow::openFileDialog()
-{
+void MainWindow::openFileDialog() {
 	const QString filePath = QFileDialog::getOpenFileName(documentArea, tr("Open BRL-CAD database"), QString(), "BRL-CAD Database (*.g)");
-    if (!filePath.isEmpty()){
+    if (!filePath.isEmpty()) {
```

Comment:
In general, you shouldn't include arbitrary white-space changes in your patch.

Reply:
I will keep that in mind. Will fix this in the next commit.